package.json

```diff
@@ -10,22 +10,20 @@
   ],
   "repository": "github:stadline/react-payline",
   "files": [
-    "**/*.js",
-    "**/*.js.map",
-    "**/*.d.ts"
+    "dist"
   ],
-  "main": "index.js",
-  "types": "index.d.ts",
+  "type": "module",
+  "exports": "./dist/index.js",
+  "types": "./dist/index.d.ts",
   "devDependencies": {
-    "@types/react": "^16.9.19",
-    "react": "^16.12.0",
-    "typescript": "^3.7.5"
+    "@types/react": "^18.0.15",
+    "react": "^18.2.0",
+    "typescript": "^4.7.4"
   },
   "peerDependencies": {
-    "react": ">16.8"
+    "react": ">=18.0"
```

Comment:
We need to support >16.8 for our app. Is there anything in current version preventing you from using the lib in a React 18 app?